doc/model_feature.md

```diff
@@ -0,0 +1,26 @@
+# Model Input Features
+The energy-related metrics collected and exported by [Kepler](https://github.com/sustainable-computing-io/kepler) are groupped by the input source as below.
+
+Group Name|Features
+---|---
+Full|WORKLOAD_FEATURES + SYSTEM_FEATURES
+WorkloadOnly|WORKLOAD_FEATURES
+CounterOnly|COUNTER_FEAUTRES
+CgroupOnly|CGROUP_FEATURES
+BPFOnly|BPF_FEATURES
+KubeletOnly|KUBELET_FEATURES
+
+*Note:* WORKLOAD_FEATURES includes COUNTER_FEAUTRES, CGROUP_FEATURES, IO_FEATURES, BPF_FEATURES and KUBELET_FEATURES.
+
+**List of exported metrics for each feature:**
+Queries: node_energy_stat, pod_energy_stat
+[metric declaration](https://github.com/sustainable-computing-io/kepler/blob/c8f0528980944d3952f001ac09cd4e285cebe73e/pkg/collector/prometheus_collector.go#L40-L80)
+
+Features|Kepler exporting labels in `pod_energy_stat` metric
+---|---
+COUNTER_FEAUTRES| - cache_miss<br>- cpu_cycles<br>- cpu_instr
```

Comment:
Correct me if incorrect, but shouldn't cpu frequency also be included?

Reply:
It should be the system feature not the counter feature. The node training should require the full feature including the system feature and the rest.